cmd/template/cluster/provider/templates/aws/template.go

```diff
@@ -0,0 +1,43 @@
+package aws
+
+import (
+	_ "embed"
+)
+
+//go:embed cluster.yaml.tmpl
+var cluster string
+
+//go:embed aws_cluster.yaml.tmpl
+var awsCluster string
+
+//go:embed kubeadm_control_plane.yaml.tmpl
+var kubeadmControlPlane string
+
+//go:embed aws_machine_template.yaml.tmpl
+var awsMachineTemplate string
+
+//go:embed aws_cluster_role_identity.yaml.tmpl
+var awsClusterRoleIdentity string
+
+//go:embed bastion_secret.yaml.tmpl
+var bastionSecret string
+
+//go:embed bastion_machine_deployment.yaml.tmpl
+var bastionMachineDeployment string
+
+//go:embed bastion_aws_machine_template.yaml.tmpl
+var bastionAWSMachineTemplate string
+
+// GetTemplate merges all .tmpl files.
+func GetTemplates() []string {
+	return []string{
```

Comment:
Ordered files for CAPA.

Reply:
Does order matter here? In that case I'd add a comment

Reply:
Makes sense to me - will add.

Reply:
Done - PTAL.